versioning.asciidoc

```diff
@@ -0,0 +1,20 @@
+= Versioning Gitenberg books
+
+Gitenberg uses http://semver.org/[Semantic Versioning] to describe the changes to books over time.
+The current version of a GITenberg book is stored in two places:
+
```

Comment:
We'll need a way to get metadata into the result files; I'm not familiar enough with ascidoctor to know what that way should be. I'm thinking about everything on the copyright page- identifiers, cataloguing data, rights notices, credits, as well as document metatags. In any case we should follow the DRY principle where that makes sense.

I don't think having everything contained in one asciidoc file is going to be sustainable, so it might be best not to rely on it at all.

I will add a commit for the dictionary with a suggested slot for version: _version